koin-projects/build.gradle

```diff
@@ -19,6 +19,9 @@ buildscript {
     }
 }
 
+wrapper {
+    distributionType = Wrapper.DistributionType.ALL
+}
```

Comment:
what is this block?

Reply:
By default, Gradle offers wrapper as BIN: `./gradlew wrapper --gradle-version = XXX`. To just copy the command to update the version, I added an indication that you must use ALL

example: https://github.com/gradle/gradle/releases/tag/v6.0.1 , section `Upgrade Instructions`

Reply:
ok